src/App.module.css

```diff
@@ -0,0 +1,102 @@
+
+/* .item {
+color: #fff;
+  font-family: "McLaren", cursive;
+  font-weight: 200;
+} */
+.nav {
+
```

Comment:
unused lines of code should be removed and code inside css parent should be indented